rust/src/ortc.rs

```diff
@@ -566,6 +572,7 @@ pub(crate) fn get_consumable_rtp_parameters(
                     rtcp_feedback: rtcp_feedback.clone(),
                 }
             }
+            _ => panic!("Unexpected codec type"),
```

Comment:
@nazar-pc, this is set here because `RtpCodecCapabilityFinalized`, which is now defined in `mediasoup-types` crate, is `non_exhaustive`.

Reply:
What is the point of making it `non_exhaustive` though? I don't think we'll have anything other than audio/video any time soon

Reply:
I think it was done so you can't construct it from externally, only using provided APIs :think

Reply:
ahh, I see. Well, members are private so they cannot be constructed externally anyway.

Reply:
> Which members are private? 

I thought for a moment that this was a struct rather than an enum.